### ISSUE TITLE
symlinks in mingw builds

### ISSUE THREAD

**Issue by krcools** [JuliaIO/LibpngBuilder#2]
It looks like the Windows binaries still have symlinks. I tried to fix it  by including a snippet like:

```
if [[ ${target} == *-w64-mingw* ]]; then
    mkdir ${WORKSPACE}/destdir/tmp
    cp -r -L ${WORKSPACE}/destdir/lib/* ${WORKSPACE}/destdir/tmp
    rm -r ${WORKSPACE}/destdir/lib
    mv ${WORKSPACE}/destdir/tmp ${WORKSPACE}/destdir/lib
    ls ${WORKSPACE}/destdir/lib
fi
```

like @bicycle1885 did to fix the libz binaries, but for now without success.

**Comment by krcools** [comment]
Hi @SimonDanisch , sorry to keep going on about this, but this stands in the way of Windows users playing around with e.g. your impressive Makie package...

**Comment by SimonDanisch** [comment]
> but for now without success.

What problem did you run into?

**Comment by krcools** [comment]
The bins generated in the new release still have symlinks in them. I tried to fix this using `cp -L` (the `-L` replaces symlinks with actual files). It took me some time to get this right because the auditing done by BinaryBuilder had messed up some of the relative paths in the symlinks, causing `cp -L` to bail.

I figured it out in the end and submitted a pull request. It runs fine on travis as launched from my fork of LibpngBuilder but for some reason it fails when launched from the PR.

See also:

https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/311

**Comment by SimonDanisch** [comment]
hm maybe it will be easier to just fix BinaryProvider instead of going through all binarybuilder repositories and waste lots of time on such details?! 
After all, those binaries with symlinks unpack perfectly fine on Julia 0.6...

cc: @staticfloat, you think it's worth our time investigating that option? If so I can continue trying to get an MWE for https://github.com/JuliaPackaging/BinaryProvider.jl/issues/86 ... or is there another fix in the pipes, that doesn't need us to fix all binarybuilders manually? https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/311 seems like that, right? Is it close to being implemented?

**Comment by simonbyrne** [comment]
BinaryBuilder has now been updated: https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/370

**Comment by SimonDanisch** [comment]
should be fixed in #6, thanks for everyone working on this!
